bin/build_test_config.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+DEST_DIR=./aws-android-sdk-testutils/src/main/res/raw
```

Comment:
Currently this script is written in such a way that it needs to be executed from the directory that owns it (use of `./`, etc.)

You could find the `$SCRIPT_PATH` with something like [this](https://stackoverflow.com/a/3572105/695787) and then `dirname $(realpath "$0")`, and base your relative paths against that.